samples/ipc/ipc_service/src/main.c

```diff
@@ -97,24 +91,18 @@ int main(void)
 
 	k_sem_take(&bound_sem, K_FOREVER);
 
-	uint16_t mlen = BASE_MESSAGE_LEN;
-
 	while (true) {
-		ret = ipc_service_send(&ep, &msg, mlen);
+		ret = ipc_service_send(&ep, p_payload, CONFIG_APP_IPC_SERVICE_MESSAGE_LEN);
 		if (ret == -ENOMEM) {
 			/* No space in the buffer. Retry. */
 			continue;
 		} else if (ret < 0) {
-			LOG_ERR("send_message(%d) failed with ret %d", msg.message, ret);
+			printk("send_message(%ld) failed with ret %d\n", p_payload->cnt, ret);
```

Comment:
maybe better use LOG_ functions instead of printk ?

Reply:
I thought we should be consistent if we use printk in other places.

Reply:
Log is just more flexible and can be configured to show for example only errors, but printk can stay, no problem.